src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,111 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+import PropTypes from 'prop-types';
 import './FindMovie.scss';
+import classNames from 'classnames';
+
+import { request } from '../../api/api';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addedMovie }) => {
+  const [foundTitle, setTitle] = useState(null);
+  const [newMovie, setMovie] = useState();
+  const [isDanger, setDanger] = useState(false);
+
+  const getMovie = async() => {
+    const movieFromServer = await request(`${foundTitle}`);
+
+    if (movieFromServer.Response !== 'False') {
+      const { Title, Plot, Poster, imdbID } = movieFromServer;
+      const movie = {
+        title: Title,
+        description: Plot,
+        imgUrl: Poster,
+        imdbUrl: `https://www.omdbapi.com/?apikey=e8bbd6b8&t=${foundTitle}`,
```

Comment:
```suggestion
        imdbUrl: ``https://www.imdb.com/title/${imdbID}/``,
```
It should be the link to IMDB site, not the api request